nixops/resources/__init__.py

```diff
@@ -73,7 +73,7 @@ def __init__(self, depl, name: str, id):
         self.logger = depl.logger.get_logger_for(name)
         self.logger.register_index(self.index)
 
-    def _set_attrs(self, attrs):
+    def _set_attrs(self, attrs) -> None:
```

Comment:
maybe this? Feeling less sure about these Any values, but the str is for sure right.

```suggestion
    def _set_attrs(self, attrs: Dict[str, Any]) -> None:
```